File: VB/13_Hyperlink/GetLinkedSlide.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation
Imports Spire.Presentation.Drawing
Imports System.IO
Imports Spire.Presentation.Charts

Namespace GetLinkedSlide
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create Presentation
			Dim presentation As New Presentation()

			'Load ppt file
			presentation.LoadFromFile("..\..\..\..\..\..\Data\linkedSlide.pptx")

			'Get the second slide
			Dim slide As ISlide = presentation.Slides(1)

			'Get the first shape of the second slide
			Dim shape As IAutoShape = TryCast(slide.Shapes(0), IAutoShape)

			'Get the linked slide index
			If shape.Click.ActionType = HyperlinkActionType.GotoSlide Then
				Dim targetSlide As ISlide = shape.Click.TargetSlide
				MessageBox.Show("Linked slide number = " & targetSlide.SlideNumber)
			End If
		End Sub
	End Class
End Namespace

File: CS/13_Hyperlink/GetLinkedSlide.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Drawing;
using System.IO;
using Spire.Presentation.Charts;

namespace GetLinkedSlide
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create Presentation
            Presentation presentation = new Presentation();

            //Load ppt file
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\linkedSlide.pptx");

            //Get the second slide
            ISlide slide = presentation.Slides[1];

            //Get the first shape of the second slide
            IAutoShape shape = slide.Shapes[0] as IAutoShape;

            //Get the linked slide index
            if (shape.Click.ActionType == HyperlinkActionType.GotoSlide)
            {
                ISlide targetSlide = shape.Click.TargetSlide;
                MessageBox.Show("Linked slide number = " + targetSlide.SlideNumber);
            }
        }
    }
}